'__optnone__' and 'always_inline' attributes are not compatible

When building on `OS X` (with clang), I get the following compilation error in `benchmark.h`

```
/src/include/OpenImageIO/benchmark.h:410:43: error: '__optnone__' and 'always_inline' attributes are not compatible
```

I am seeing this with `oiio 1.8.6` (however the code in question does not appear to have changed recently). EDIT: I also see the issue in 1.8.8

Here is the issue that I am seeing
```
[ 56%] Building CXX object src/libOpenImageIO/CMakeFiles/OpenImageIO.dir/__/webp.imageio/webpoutput.cpp.o
In file included from /Users/travis/miniconda3/conda-bld/openimageio_1520955523932/work/oiio-Release-1.8.6/src/tiff.imageio/tiffoutput.cpp:54:
In file included from /Users/travis/miniconda3/conda-bld/openimageio_1520955523932/work/oiio-Release-1.8.6/src/include/OpenImageIO/timer.h:271:
/Users/travis/miniconda3/conda-bld/openimageio_1520955523932/work/oiio-Release-1.8.6/src/include/OpenImageIO/benchmark.h:410:43: error: '__optnone__' and 'always_inline' attributes are not compatible
OIIO_FORCEINLINE T const & __attribute__((__optnone__))
```

--- a/src/include/OpenImageIO/benchmark.h
+++ b/src/include/OpenImageIO/benchmark.h
@@ -443,7 +443,7 @@ OIIO_FORCEINLINE T const & DoNotOptimize (T const &val) {
 // If __optnone__ attribute is available: make a null function with no
 // optimization, that's all we need.
 template <class T>
-OIIO_FORCEINLINE T const & __attribute__((__optnone__))
+inline T const & __attribute__((__optnone__))
 DoNotOptimize (T const &val) {
     return val;
 }

